src/components/Button/NavigateButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from '@emotion/native';
+import {Text} from '../Text';
+import {Icon} from '../Icon';
+import {arrowRightXmlData} from '../../assets/svg';
+import {Tpalette} from '../../assets/styles/emotion';
+
+export interface INavigateButtonProps {
+  text: string;
+  width?: string;
+  color?: Tpalette;
+  backgroundColor?: Tpalette;
+  onPress: () => {};
```

Comment:
```suggestion
  onPress: () => void;
```
위에서 작성해주신 것 처럼 void를 반환해도 좋을 것 같습니다!